### DISCUSSION TITLE
PDF License Auto Send / Change Title

### DISCUSSION THREAD

**Discussion by yaarite** [gdarko/digital-license-manager#40] [General]
This plugin is wonderful and does almost everything I needed. I am using it to generate Warranties for a product (re-purposed) because there is no warranty registration plugins that are this efficient. The following 2 suggestions would be a relatively easy fix IMO. 

1) When a customer places an order for a Digital license they should not only get a copy of the order but also emailed a copy of the PDF license or at the very least a include a download link in the woocommerce order. Rather than having them log into the wordpress site after and manually download the license  from the link inside their account (way to many additional steps) . 

2) Also one more simple functionality in addition to the current Logo branding feature would be the ability to actually re-name the PDF or license Title . This would be a welcome additional functionality to white label the document. Instead of the word License at the top of the Document I would  like to be able to change it to Warranty or Prefix it with the Product name. 

At the very least tell us what file to edit in the plugin to accomplish the PDF Title Change

**Comment by gdarko** [comment]
Hey @yaarite  - This is answered here
https://github.com/gdarko/digital-license-manager/discussions/41
